open_seq2seq/losses/tacotron_loss.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2018 NVIDIA Corporation
+
+from __future__ import absolute_import, division, print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import tensorflow as tf
+
+from .loss import Loss
+
+class TacotronLoss(Loss):
+  def __init__(self, params, model, name="cross_entropy_loss"):
+    super(TacotronLoss, self).__init__(params, model, name)
+
+  def get_optional_params(self):
+    """Static method with description of optional parameters.
+
+      Returns:
+        dict:
+            Dictionary containing all the parameters that **can** be
+            included into the ``params`` parameter of the
+            class :meth:`__init__` method.
+    """
+    return {
+      'use_mask': bool,
+    }
+
+  def _compute_loss(self, input_dict):
```

Comment:
Can you check the codestyle for this function to be compliant with PEP-8 aside from using 2 spaces?

Reply:
And also for other functions in other files